System > Common Good Parachains

joepetrowski opened the pull request.

joepetrowski committed:
rename file

joepetrowski committed:
use system over common good

profradha commented on docs/build/build-hrmp-channels.md:
```diff
@@ -28,23 +28,22 @@ In order to do this, the calls to be dispatched from both chains are:
-In order to provide complete contextual information and transparency to the community, which is the
-ultimate decision maker of the proposals, we encourage following the process implemented on Kusama.
+Opening an HRMP channel with a system parachain requires a referendum. Like all other governance
+proposals, proposers should follow best practices like opening a discussion on Polkassembly or
```
```suggestion
proposals, proposers should follow best practices like opening a discussion on [Polkassembly](https://polkadot.polkassembly.io/) or
```

profradha commented on docs/build/build-hrmp-channels.md:
```diff
@@ -28,23 +28,22 @@ In order to do this, the calls to be dispatched from both chains are:
-ultimate decision maker of the proposals, we encourage following the process implemented on Kusama.
+Opening an HRMP channel with a system parachain requires a referendum. Like all other governance
+proposals, proposers should follow best practices like opening a discussion on Polkassembly or
+Subsquare and then submitting the proposal on-chain.
```
```suggestion
[Subsquare](https://polkadot.subsquare.io/) and then submitting the proposal on-chain.
```

profradha commented on docs/learn/learn-parachains.md:
```diff
@@ -169,16 +169,15 @@ maintain the stronger property of single block composability.
-utility chains. These typically do not have an economic model and help remove transactions from the
-Relay Chain, allowing for more efficient parachain processing.
+[governance](learn-governance.md) system, and are deemed as part of the network's protocol, such as
+bridges to other networks or chains. These typically do not have an economic model and help remove\
```
```suggestion
bridges to other networks or chains. These typically do not have an economic model and help remove 

```

DrW3RK committed:
Implement redirect

Ensure the common good parachains Wiki doc links on third party websites work well

DrW3RK reviewed (approved):
Looks good to me! Implemented redirect for the common good parachain doc to the the newly created systems parachain doc, which ensures the wiki links on third party websites are not broken. 

Also, IMO Encointer seems to not fit the template definition of a system parachain.

joepetrowski committed:
Apply suggestions from code review

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>
